src/platform/forms-system/src/js/components/ConfirmationView/ConfirmationView.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
```

Comment:
This file migrated from 
https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/simple-forms/shared/components/ConfirmationPageView.v2.jsx. There are major changes here, including breaking out all components into ./components file, and using a subcomponent pattern with useContext